### PR TITLE
Clang 8

### PR DESCRIPTION
**Pull request by achamayou**

### CODE REVIEW

**Comment by cimetrics** [comment]
images

File: tests/coverage/generate_coverage.sh [hunk 1]
```diff
@@ -6,27 +6,31 @@ set -ex
 
 suffix=$1
 
+LLVM_VER=7
```
Comment (olgavrou) [inline tests/coverage/generate_coverage.sh]: 7 or 8?

Comment (achamayou) [reply tests/coverage/generate_coverage.sh]: Trying to get this from CMake, to keep it in sync/compatible.

**Comment by cimetrics** [comment]
images

**Comment by achamayou** [comment]
Single test failure solved by #446, so merging anyway

**Pull request merged**